homeassistant/components/http/__init__.py

```diff
@@ -240,6 +246,40 @@ def log(
         super().log(request, response, time)
 
 
+class HomeAssistantRequest(web.Request):
```

Comment:
this results in a deprecation warning in tests
```
../../usr/local/lib/python3.10/abc.py:106
  /usr/local/lib/python3.10/abc.py:106: DeprecationWarning: Inheritance class HomeAssistantApplication from web.Application is discouraged
```
inheritance has been deprecated in aiohttp with https://github.com/aio-libs/aiohttp/pull/2692 and will be removed in aiohttp 4.0.0 with https://github.com/aio-libs/aiohttp/pull/3933

Reply:
See https://github.com/home-assistant/core/pull/90214#discussion_r1147809270

Reply:
ah sorry, did somehow oversee this comment 🙈